tests/unispeech_sat/test_modeling_unispeech_sat.py

```diff
@@ -865,10 +857,10 @@ def test_inference_encoder_large(self):
 
         # fmt: off
         expected_hidden_states_slice = torch.tensor(
-            [[[-0.1172, -0.0797],
```

Comment:
This change was incorrect and due to a local mismatch of the Datasets library. Need to be more careful when correcting hard-coded tests the next time.